spring-cloud-aws-samples/spring-cloud-aws-parameter-store-sample/src/main/resources/application.properties

```diff
@@ -1,5 +1,6 @@
 # importing single parameter
-spring.config.import=aws-parameterstore:/config/spring/
+#spring.config.import=aws-parameterstore:/config/spring/
+logging.level.io.awspring.cloud=debug
```

Comment:
We should add the same line to the Secrets Manager sample as well

Reply:
good catch